velox/dwio/parquet/writer/Writer.h

```diff
@@ -117,6 +124,10 @@ struct WriterOptions : public dwio::common::WriterOptions {
       "hive.parquet.writer.timestamp_unit";
   static constexpr const char* kParquetHiveConnectorWriteTimestampUnit =
       "hive.parquet.writer.timestamp-unit";
+  static constexpr const char* kParquetSessionDataPageVersion =
+      "parquet_datapage_version";
+  static constexpr const char* kParquetHiveConnectorDataPageVersion =
+      "hive.parquet.datapage.version";
```

Comment:
These should be `hive.parquet.writer.datapage_version` and `hive.parquet.writer.datapage-version` to be consistent with the timestamp-unit above.

Reply:
Ah right, I didn't catch that either. Had `parquet_datapage_version` without hive prefix as that was the format of the prior value being picked up from the config. 

Fixed the naming.